### PR TITLE
Add custom padding to Modal component

### PR DESCRIPTION
**Pull request by sun-tea**
- [ ] Feature
- [ ] Fix
- [x] Enhancement

## Todo - Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.

### CODE REVIEW

**Comment by sun-tea** [comment]
@spartDev I was wondering why the snapshot was already empty for `should render open modal`? Shouldn't that be the opposite?

File: src/Modal/__stories__/Modal.stories.js [hunk 1]
```diff
@@ -56,6 +67,7 @@ storiesOf('Modal', module)
             active={store.get('active')}
             width={`${widthValue.toString()}rem`}
             height={`${heightValue.toString()}rem`}
+            padding={`${paddingValue.toString()}rem`}
```
Comment (spartDev) [inline src/Modal/__stories__/Modal.stories.js]: You can remove `toString()`, all values passed in `` are transformed to string

File: src/Modal/__stories__/Modal.stories.js [hunk 2]
```diff
@@ -100,6 +123,7 @@ storiesOf('Modal', module)
           active={activeValue}
           width={`${widthValue.toString()}rem`}
           height={`${heightValue.toString()}rem`}
+          padding={`${paddingValue.toString()}rem`}
```
Comment (spartDev) [inline src/Modal/__stories__/Modal.stories.js]: Same here

File: src/Modal/__stories__/Modal.stories.js [hunk 3]
```diff
@@ -163,6 +198,7 @@ storiesOf('Modal', module)
             active={store.get('active')}
             width={`${widthValue.toString()}rem`}
             height={`${heightValue.toString()}rem`}
+            padding={`${paddingValue.toString()}rem`}
```
Comment (spartDev) [inline src/Modal/__stories__/Modal.stories.js]: same here

**Pull request merged**